### DISCUSSION TITLE
I receive GET requests on my localhost:3000 from socket.io, but I've never used it

### DISCUSSION THREAD

**Discussion by B3rC** [socketio/socket.io#5060] [Q&A]
Hey! If I run something on localhost:3000, I always get these socket.io requests on my terminal:
![Képernyőkép 2024-07-04 183928](https://github.com/socketio/socket.io/assets/42650162/0815a207-8dc0-4c13-a404-18555ee228a3)
I don't know what can be the reason, I've never even heard about socket.io, thought maybe someone could help. It's probably coming from my computer, but I have no idea from where, as I mentioned, I never used socket.io, didn't imported it or downloaded it. The issue occurs only on localhost:3000, other ports not affected

**Comment by darrachequesne** [comment]
Hi! That's weird indeed. Did you check if the HTTP request comes with an `origin` header?
